inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -654,12 +641,6 @@
                         "description": "Just in case...",
                         "title": "Other IDs"
                     },
-                    "uuid": {
-                        "title": "UUID",
-                        "description": "Universally unique identifier of the author.",
-                        "type": "string",
-                        "pattern": "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"
-                    },
```

Comment:
Actually this should be in the schema. It should not be touched by catalogers though. This is added on `before_record_update`/`_inser`, so it goes directly into the DB.